source/server/configuration_impl.cc

```diff
@@ -86,6 +86,16 @@ void MainImpl::initialize(const envoy::config::bootstrap::v3::Bootstrap& bootstr
   stats_flush_interval_ =
       std::chrono::milliseconds(PROTOBUF_GET_MS_OR_DEFAULT(bootstrap, stats_flush_interval, 5000));
 
+  switch (bootstrap.stats_flush_case()) {
+  case envoy::config::bootstrap::v3::Bootstrap::kStatsFlushOnAdmin: {
+    stats_flush_on_admin_ = bootstrap.stats_flush_on_admin();
+    break;
+  }
+  case envoy::config::bootstrap::v3::Bootstrap::STATS_FLUSH_NOT_SET: {
+    break; // do nothing
```

Comment:
If nothing is done here, why not use an if clause instead of switch?

Reply:
if statement makes the most sense here